src/platform/mock-local-home-platform.ts

```diff
@@ -156,8 +156,8 @@ export class MockLocalHomePlatform implements MockUDPListener {
 
     const device = identifyResponse.payload.device;
     console.log('Registering localDeviceId: ' + device.verificationId);
-    this.localDeviceIds.set(device.id, device.verificationId);
-    this.onNewDeviceIdRegistered(device.verificationId);
+    this.localDeviceIds.set(device.id, device.verificationId!);
```

Comment:
Is `device.verificationId` sometimes undefined?

Reply:
I added an early exit if `device.verificationId` is null.  I'll double check this is the expected behavior.

Reply:
See #32